packages/net-stubbing/lib/server/intercept-request.ts

```diff
@@ -194,12 +194,6 @@ function _interceptRequest (state: NetStubbingState, request: BackendRequest, ro
     emit(socket, 'http:request:received', frame)
   }
 
-  if (route.staticResponse) {
-    emitReceived()
-
-    return sendStaticResponse(request.res, route.staticResponse, request.onResponse!)
-  }
-
   const ensureBody = (cb: () => void) => {
```

Comment:
right now every branch of the code that follows this declaration is using both `ensureBody` & `emitReceived`, so maybe you'd like to have this refactored somehow? If yes - let me know if you have anything in particular in mind. A change is not needed that much as the code is readable but maybe it would be good to ensure that both of those things will always be called for this logic rather if some new branches get added?